Zend/tests/bug46106.phpt

```diff
@@ -23,5 +23,5 @@ try {
 ?>
 DONE
 --EXPECT--
-str_pad() expects at least 2 parameters, 1 given
+str_pad(): At least 2 arguments are expected, 1 given
```

Comment:
parameters -> arguments here make sense, but I liked the general phrasing more in the previous variant.

Reply:
I understand it (even I wasn't 100% sure), so I can revert to the original wording, except for using parameter(s).